dts/bindings/rtc/nxp,irtc.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: IRTC
+
+compatible: "nxp,irtc"
+
+include:
+  - name: rtc.yaml
+  - name: rtc-device.yaml
+
+properties:
+  reg:
+    required: true
+
+  clock-src:
+    type: int
+    required: true
+    enum:
+      - 0
+      - 1
+    description: |
+      The input clock select for IRTC.
+      0 <- 16.384 kHz
+      1 <- 32.768 kHz
+
+  output-clk-en:
+    type: int
+    default: 0
+    description: |
```

Comment:
`enum:` here as well for the 4 supported values :)

Reply:
Added enum field, kept the value to 0-3 so they can be pasted in at build time in INIT macro